airbyte-webapp/src/components/connection/ConnectionInfoCard/ConnectionInfoCard.test.tsx

```diff
@@ -7,7 +7,7 @@ import { ConnectionStatus, SchemaChange } from "core/request/AirbyteClient";
 import { defaultOssFeatures, FeatureItem } from "hooks/services/Feature";
 
 // eslint-disable-next-line css-modules/no-unused-class
-import styles from "./StatusMainInfo.module.scss";
+import styles from "./connectionInfo.module.scss";
```

Comment:
I think this import should be from `"./ConnectionInfoCard.module.scss"`?

Reply:
Whoops, bad rename. Fixed